modules/helm-initializer/main.tf

```diff
@@ -61,9 +56,9 @@ resource "null_resource" "install_tiller" {
         --service-account tiller \
         --tiller-tls \
         --tiller-tls-verify \
-        --tls-ca-cert=${local_file.ca_cert.filename} \
-        --tiller-tls-cert=${local_file.tiller_cert.filename} \
-        --tiller-tls-key=${local_file.tiller_key.filename} \
+        --tls-ca-cert=${local.tls_dir}/ca.cert.pem \
+        --tiller-tls-cert=${local.tls_dir}/tiller.cert.pem \
+        --tiller-tls-key=${local.tls_dir}/tiller.key.pem \
```

Comment:
What about keeping the actual filenames as variables? -> They would be potentially customisable + we would avoid repeating hardcoded names at 3 diff places.

Reply:
Sounds good!